### ISSUE TITLE
[Translation] Update Ukrainian translations

### ISSUE THREAD

**Issue by argasek** [KoalicjaOtwartyKrakow/frontend#51]

**Comment by koral--** [comment]
It will be more convenient to use some Localization as a service.
Non-technical translators can easily provide translations on some web-based UI and there will be one source of truth for all the platforms.
There is a LaaS provided by i18next authors: https://locize.com/

For now I've just copy-pasted jsons from here into flutter app repo.

**Comment by argasek** [comment]
@koral-- I agree. They are in state of flux however, plus I suspect there will be some disparity anyway thanks to the mobile nature and different audience of the frontend & mobile apps. But that's definitely something we should think of in the future. Calling @unjello to make a note, as this is more like architecture / infrastructure task, which probably will also touch backend.
